bus-mapping/src/evm/opcodes.rs

```diff
@@ -129,7 +131,7 @@ fn fn_gen_associated_ops(opcode_id: &OpcodeId) -> FnGenAssociatedOps {
         OpcodeId::NUMBER => StackOnlyOpcode::<0, 1>::gen_associated_ops,
         // OpcodeId::DIFFICULTY => {},
         // OpcodeId::GASLIMIT => {},
-        // OpcodeId::CHAINID => {},
+        OpcodeId::CHAINID => ChainId::gen_associated_ops,
```

Comment:
I think ChainId can be a `StackOnlyOpcode<0, 1>`?

Reply:
Agree, changed!